ports/unix/README.md

```diff
@@ -1,14 +1,49 @@
-The Unix version
-----------------
+# The Unix port
 
-The "unix" port requires a standard Unix-like environment with gcc and GNU
-make. This includes Linux, BSD, macOS, and Windows Subsystem for Linux. The
+The "unix" port requires a standard Unix-like environment with gcc, GNU
+make, and Python 3.x. This includes Linux, BSD, macOS, and Windows Subsystem for Linux. The
 x86 and x64 architectures are supported (i.e. x86 32- and 64-bit), as well as
 ARM and MIPS. Making a full-featured port to another architecture requires
 writing some assembly code for the exception handling and garbage collection.
 Alternatively, a fallback implementation based on setjmp/longjmp can be used.
 
-To build (see section below for required dependencies):
+# Building
+
+## Dependencies
+
+To build the "minimal" variant you will need:
+
+* A git client, and a local git clone of the MicroPython source
```

Comment:
Do you strictly need git?  If you download the source from GitHub or at the top of https://micropython.org/download/ then I don't think you need git.

Reply:
Good question... I did a few more quick tests, and it breaks down as:

* The instructions say to run the `make submodules` target, and that needs git.
* If you build from a release .tar.xz and skip the submodules step then you don't need git
* If you download direct from GitHub then you can build minimal variant but otherwise it won't work without the submodules.

I'll make some edits to try and communicate this without it being too fiddly.

Reply:
How's that look?

Reply:
Yes, that looks fine now.